tests/scripts/thread-cert/border_routing/test_multi_thread_networks.py

```diff
@@ -119,6 +119,25 @@ def test(self):
         self.assertTrue(len(self.nodes[BR2].get_prefixes()) == 1)
         self.assertTrue(len(self.nodes[ROUTER2].get_prefixes()) == 1)
 
+        br1_omr_prefix = self.nodes[BR1].get_prefixes()[0]
+        br2_omr_prefix = self.nodes[BR2].get_prefixes()[0]
+
+        self.assertNotEqual(br1_omr_prefix, br2_omr_prefix)
+
+        # Each BR should independently register an external route for the on-link prefix
+        # and OMR prefix in another Thread Network.
+        self.assertTrue(len(self.nodes[BR1].get_routes()) == 2)
+        self.assertTrue(len(self.nodes[ROUTER1].get_routes()) == 2)
```

Comment:
nit: update Line#43 in topology `ED` -> `ROUTER` to match the device type in use